Misc/NEWS.d/next/Library/2020-04-12-02-46-29.bpo-40221.rQmSte.rst

```diff
@@ -0,0 +1 @@
+Update ForkAwareThreadLock to use _at_fork_reinit.
```

Comment:
For such news entry, you should mention which module is modified. The user reading the NEWS entry has no idea, except maybe multiprocessing experts ;-)

For this specific change, honestly, I don't think that end users really care: I suggest to remove the NEWS entry. The change doesn't seem to fix any known crash, it's more a cleanup/refactoring.